<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#Form
{
	background-color: rgb(249, 249, 249);
}


QWidget
{
	background-color: rgba(239, 239, 239, 0);
}

QLabel
{
	padding: 2px;
}

QPushButton 
{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(200, 200, 200, 255), 
stop:0.5 rgba(221, 221, 221, 255),
stop:1 rgba(200, 200, 200, 255));
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255)
 }

QPushButton#pushButton_02_buyitnow
{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(234, 0, 0, 255), 
stop:0.5 rgba(255, 39, 39, 255),
stop:1 rgba(234, 0, 0, 255));
	color: rgb(249, 249, 249);
}

 QPushButton:pressed {
	background-color: rgb(221, 221, 221);
 }

 QPushButton:hover {
	 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(229, 229, 229, 255), 
stop:0.5 rgba(221, 221, 221, 255),
stop:1 rgba(229, 229, 229, 221));
    border-color: rgba(221, 128, 32, 255)
 }


QLineEdit
{
	background-color:  rgba(230, 230, 230, 255);
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255);
 }
</string>
  </property>
  <widget class="QLabel" name="label_01_image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;Image&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_01_addimage">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_02_deleteimage">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_03_sell">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sell</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_thumbnail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_04_cancel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_02_name">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_03_buyoutprice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buyout Price</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_04_bidprice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start Bid Price</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_02_name">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_03_buyoutprice">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_04_bidprice">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_buyoutavailable">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>50</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_06_description">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_06_description">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_05_categories">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Categories</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_05_categories">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_02_name</tabstop>
  <tabstop>checkBox_buyoutavailable</tabstop>
  <tabstop>lineEdit_03_buyoutprice</tabstop>
  <tabstop>lineEdit_04_bidprice</tabstop>
  <tabstop>lineEdit_05_categories</tabstop>
  <tabstop>textEdit_06_description</tabstop>
  <tabstop>listWidget_thumbnail</tabstop>
  <tabstop>pushButton_01_addimage</tabstop>
  <tabstop>pushButton_02_deleteimage</tabstop>
  <tabstop>pushButton_03_sell</tabstop>
  <tabstop>pushButton_04_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
